src/officehours_api/models.py

```diff
@@ -31,6 +40,7 @@ class Profile(models.Model):
     phone_number = models.CharField(max_length=20, default="", blank=True, null=False)
     notify_me_attendee = models.BooleanField(default=False)
     notify_me_host = models.BooleanField(default=False)
+    backend_metadata = JSONField(null=True, default=dict)
```

Comment:
Did you have something in mind? I thought along similar lines, but ultimately kept `backend_metadata` since `profile_backend_metadata` would be redundant - I can already see that it's for the `Profile`. I didn't want to tie it to Zoom either (i.e. `zoom_metadata`) since these are meant to be future proofed against possible future backends.

Reply:
Yeah, I'd maybe go with `backend_account_metadata`, since that's my understanding of it? That's also kinda long, so I'm fine with keeping it as if you'd prefer.